packages/vite-plugin/src/client/iife/content-pro-loader.ts

```diff
@@ -1,4 +1,6 @@
 declare const __SCRIPT__: string
+
+localStorage.setItem('__CRX_EXTEND_ID__', chrome.runtime.id)
```

Comment:
did you try to obtain it using the [method described here](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/Hs1X-Zf3ze0)  by the chromium team? It would be cool to not rely on localStorage

```js
// your injected.js. Only works as a first line in the injected script
extensionID = document.currentScript.id;
```

If not then lets try that